app/src/main/java/timur/gilfanov/messenger/ui/screen/user/LanguageViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package timur.gilfanov.messenger.ui.screen.user
+
+import androidx.lifecycle.ViewModel
+import org.orbitmvi.orbit.Container
+import org.orbitmvi.orbit.ContainerHost
+
+class LanguageViewModel :
+    ViewModel(),
+    ContainerHost<LanguageUiState, LanguageSideEffects> {
+    override val container: Container<LanguageUiState, LanguageSideEffects>
+        get() = TODO("Not yet implemented")
+}
```

Comment:
Must add a method to change an UI language:
```suggestion
    
    fun changeLanguage(newLanguage: LanguageItem): Unit = TODO("Not yet implemented")
}
```